### DISCUSSION TITLE
How can I access the GUI app on Android TV?!

### DISCUSSION THREAD

**Discussion by hamedsbt** [mywalkb/LSPosed_mod#135] [Q&A] [answered]
Hi, I've installed the LSPosed Magisk module and rebooted my Android TV, but I can't access the LSPosed GUI app. How can I access the GUI app? Can I access it using an ADB command?

**Comment by hamedsbt** [comment, marked as answer]
SOLUTION:
Install manager app from: ```/data/adb/modules/zygisk_lsposed/manager.apk```

**Reply by joaocandre** [reply]
I can't find that apk (nor the '/data/adb' directory).
